src/main/java/core/basesyntax/service/impl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvFileReaderServiceImpl implements CsvFileReaderService {
```

Comment:
```suggestion
public class ReaderImpl implements Reader {
```
be consistent with your names